packages/docusaurus-theme-classic/src/theme/DocItem/index.js

```diff
@@ -75,7 +75,11 @@ function DocItem(props) {
     },
   } = DocContent;
 
-  const metaImageUrl = siteUrl + useBaseUrl(metaImage);
+  const _metaImageUrl = siteUrl + useBaseUrl(metaImage);
+  const externalRegex = /^(https?:|\/\/)/;
```

Comment:
One moment - let's use  `isInternalUrl`  util function instead of this regex, and also rewrite the code like this:

```js
import isInternalUrl from '@docusaurus/isInternalUrl';

let metaImageUrl = siteUrl + useBaseUrl(metaImage);

if (!isInternalUrl(metaImage)) {
  metaImageUrl = metaImage
}
```

Reply:
@lex111 
Thank you for pointing out.
I changed this.